### ISSUE TITLE
Correct/Update documentation

### ISSUE THREAD

**Issue by charbull** [MakoLab/RomanticWeb#66]
Hello,

With the RomatincWeb version 1.0.0-rc1, if there is ICollection or IList the annotation should state "Collection" instead of "Property" in the documentation?
http://romanticweb.net/docs/basic-usage/mapping/

using RomanticWeb.Mapping.Attributes;
using RomanticWeb.Mapping.Model;

public interface IPerson : IEntity
{
    [Property("http://xmlns.com/foaf/0.1/givenName")]
    string Name { get; set; }

    [Property("http://xmlns.com/foaf/0.1/familyName")]
    string LastName { get; set; }

    [Property("http://xmlns.com/foaf/0.1/knows")]
    ICollection<IPerson> Friends { get; set; }
}

**Comment by alien-mcl** [comment]
Yes indeed, this is an obvious error in the example. I'll try to have that fixed ASAP.
